index.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+require 'src/db.php';
+
+require 'src/functions.php';
+
+?>
+
+<!DOCTYPE html>
+<html lang="en" lang="vi">
```

Comment:
I wonder what consequence two languages will have...?

Reply:
just looked it up here https://stackoverflow.com/questions/12150369/define-multiple-languages-in-html-root-elements-lang-attribute 

gonna delete the lang="vi"